8.0/Dockerfile.fedora

```diff
@@ -36,16 +36,12 @@ LABEL summary="${SUMMARY}" \
 
 EXPOSE 3306
 
-# On Fedora, we fake missing python binary. In case user installs the python2
-# in the container, this hack will be removed by installing /usr/bin/python from RPM.
-RUN ln -s /usr/bin/python3 /usr/bin/python
-
 # This image must forever use UID 27 for mysql user so our volumes are safe in the future. This should *never* change.
 # Instead of relying on the DB server package, we will do the setup ourselves before any package is installed
 RUN /usr/sbin/groupadd -g 27 -o -r mysql && \
     /usr/sbin/useradd -M -N -g mysql -o -r -d ${HOME} -s /sbin/nologin -c "MySQL Server" -u 27 mysql && \
     test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)" && \
-    INSTALL_PKGS="rsync tar gettext hostname bind-utils groff-base policycoreutils" && \
+    INSTALL_PKGS="rsync tar gettext hostname bind-utils groff-base policycoreutils python-unversioned-command" && \
```

Comment:
Do we need unversioned Python in DB container image? I mean, if the DB itself needs it, it should depend on it on RPM level. Otherwise, the best practice is to use the versioned command to be sure you run the version you want every time.

Reply:
The DB itself does not need it.

I checked the codebase and I haven't found any other part of the container that would depend on the unversioned command.

Furthermore, the `python-unversioned-command` package is already part of the `s2i-core` image, which require `python3` which brings the `python-unversioned-command` as a weak dependency.

I updated the PR and removed the installation of the `python-unversioned-command` in this image.